autopush/db.py

```diff
@@ -21,6 +22,51 @@
 log = logging.getLogger(__file__)
 
 
+def next_month(delta=1):
```

Comment:
It looks like you never call next_month(); there's always a parameter. Calling functions default delta to 0. Should we be consistent here?

Also minor nit: might want to rename this get_month since it tends to either return the current, or previous month.